x-pack/plugins/cases/public/components/connectors/types.ts

```diff
@@ -45,9 +46,11 @@ export interface CaseConnectorsRegistry {
   list: () => CaseConnector[];
 }
 
-export interface ConnectorFieldsProps<TFields> {
-  isEdit?: boolean;
+export interface ConnectorFieldsProps {
+  connector: CaseActionConnector;
+}
+
+export interface ConnectorFieldsPreviewProps<TFields> {
```

Comment:
Two components, one for previewing the fields and one for the forms. The form does not need the fields. It will get it through the form context provided by the form lib.